### DISCUSSION TITLE
[Dropdown] Fixed option for search dropdown

### DISCUSSION THREAD

**Discussion by vincenccee** [fomantic/Fomantic-UI#896] [Modules (Modal, Dropdown, Popup, Calendar, Toast, ...)] [answered]
# Help Wanted
I need some help to create an option in the dropdown that (i) is fixed in the last position, (ii) is not filtered while searching and (iii) keeps the keyboard navigation to select. The following gif shows exactly the behavior that I’m looking for (it was implemented using select2 library):

![select_add_option](https://user-images.githubusercontent.com/8618475/61739431-c5e48980-ad62-11e9-958f-778afc862ad3.gif)

## Problem
Here is what I got so far:
Using the built-in addition feature with these two settings: `{ allowAdditions: true ,hideAdditions: false }`, I was able to create an option with the ability to be selected using the keyboard keys navigation that doesn't get filtered, but I'm unable to always display this fixed `addition` option in the list. The `addition` item is only displayed after the user starts typing to search. Another problem here is the fact that I can't place this `addition` option at the end of my items list.
These are the settings that I am using for the addition version:
```
$(this._elm).dropdown({
ignoreDiacritics: true,
fullTextSearch: 'exact',
ignoreCase: true,
selectOnKeydown: false,
forceSelection: false,
allowAdditions: true,
hideAdditions: false,
message: {
addResult: this.addResultsMessage()
}
});
```
And this is how it's looking like:

![fomantic_add_option_addition](https://user-images.githubusercontent.com/8618475/61739452-d0068800-ad62-11e9-835b-16dfae187870.gif)

I also tried another approach by creating an item with a `display: block !important` to make sure that is always displayed, but after I start to type to search this option gets filtered and I lose the ability to navigate with the keyboard keys:

![fomantic_add_option_fixed](https://user-images.githubusercontent.com/8618475/61739459-d39a0f00-ad62-11e9-82de-e808102c1a58.gif)

On this gif I always tried to select the add option with the keybord keys.

**Comment by maidzen** [comment]
can you recreate your last example in jsfiddle?

Here is a fomantic sandbox to fork
https://jsfiddle.net/31d6y7mn

**Comment by vincenccee** [comment]
Here it is: https://jsfiddle.net/5ehm67cp/

If you have any more questions about the problem let me know!

**Comment by maidzen** [comment, marked as answer]
i don't think fomantic Dropdown API can do this right now so i would handle it in a custom event

https://jsfiddle.net/a51zLeof/

**Comment by maidzen** [comment]
hello @vincenccee did my fiddle work out for you ? do you need any more help? 

if not please close the issue :)

**Comment by vincenccee** [comment]
yes, it is working! thanks for the help @maidzen 😄

**Reply by ivangomes** [reply]
Fala Vincee tudo bem? Você chegou a implementar a funcionalidade de adicionar a nota utilizando o botão?

**Comment by lubber-de** [comment]
Meanwhile this functionality is inbuild the nightly 2.9.0 using `unfilterable actionable item`
See https://jsfiddle.net/lubber/ar1pL0dv/5/
